packages/cli/lib/ui.js

```diff
@@ -1,6 +1,16 @@
 const chalk = require('chalk');
 const supportsHyperlinks = require('supports-hyperlinks');
 const { getAccountConfig } = require('@hubspot/cli-lib/lib/config');
+const { logger } = require('@hubspot/cli-lib/logger');
+
+/**
+ * Outputs horizontal line
+ *
+ * @returns
+ */
+const horizontalLine = () => {
```

Comment:
This all looks great to me 👍 

I wonder if it's worth trying to standardize the naming of these utils a little bit to make their purpose more clear. Not a blocker though, just an idea.

Reply:
Are you thinking along the lines of `uiLink` and `uiLine` or something else?

Reply:
Yeah exactly, just something that lets the user know that it's a UI util.